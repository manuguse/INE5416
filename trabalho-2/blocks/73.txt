1 1 7 7 7 12 12 20 20 20
1 1 1 7 7 13 13 21 20 20
1 5 5 5 7 13 15 16 16 16
1 2 8 8 8 13 16 16 17 22
2 2 6 8 8 8 17 17 17 22
2 6 6 8 11 11 18 18 22 22
3 3 3 9 11 11 18 18 18 18
3 3 9 9 10 14 14 14 18 23
3 4 9 9 10 14 19 14 23 23
4 4 9 10 10 10 14 14 23 23