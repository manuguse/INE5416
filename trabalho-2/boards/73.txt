- - - 4 6 1 - 5 - 2
- - 6 1 - - 4 - - -
- - 1 - - - - - 2 3
2 - 7 4 - - 4 - 3 -
- 1 - - 1 5 - - - 3
2 - 1 - - 4 2 6 - -
- - - 3 - - - - 5 -
2 4 - - 4 - - - - -
- - - 1 - - - 5 - 2
2 - - - - 1 7 - 3 1