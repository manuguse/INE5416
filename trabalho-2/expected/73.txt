7 3 2 4 6 1 2 5 4 2
5 1 6 1 5 3 4 1 3 1
4 3 1 2 3 2 1 5 2 3
2 4 7 4 6 1 4 1 3 4
3 1 2 3 1 5 1 4 2 3
2 3 1 2 3 4 2 6 1 2
3 6 5 3 1 2 1 4 5 7
2 4 6 2 4 3 4 6 3 5
1 3 5 1 3 2 1 5 4 2
2 1 4 5 2 1 7 1 3 1